components/SwiperSlider.js

```diff
@@ -0,0 +1,59 @@
+import { Swiper, SwiperSlide } from "swiper/react";
+import { FreeMode, Pagination, Autoplay } from "swiper/modules";
+import "swiper/css";
+import "swiper/css/free-mode";
+import "swiper/css/pagination";
+
+import Image from "next/image";
+import styled from "styled-components";
+
+export default function SwiperSlider() {
+  const images = [
+    "/images/wave1.jpg",
+    "/images/wave2.jpg",
+    "/images/wave3.jpg",
+  ];
+  return (
+    <StyledSlider>
+      <Swiper
+        modules={[FreeMode, Pagination, Autoplay]}
+        spaceBetween={10}
+        slidesPerView={1}
+        freeMode={true}
+        pagination={{ clickable: true, type: "bullets" }}
+        autoplay={{
+          delay: 5000,
+          disableOnInteraction: false,
+        }}
+      >
+        {images.map((src, index) => (
+          <SwiperSlide key={index}>
```

Comment:
never use the index of an array item as the key. either use the image url itself or define an Id for each image element.

Reply:
ohh another learning! Thanks again for reminding me @F-Kirchhoff. I added an id and access it in the mapping!